src/plugins/share/public/components/tabs/embed/index.tsx

```diff
@@ -8,55 +8,25 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import React, { useCallback } from 'react';
+import React from 'react';
 import { type IModalTabDeclaration } from '@kbn/shared-ux-tabbed-modal';
 import { EmbedContent } from './embed_content';
 import { useShareTabsContext } from '../../context';
 
-const EMBED_TAB_ACTIONS = {
-  SET_EMBED_URL: 'SET_EMBED_URL',
-  SET_IS_NOT_SAVED: 'SET_IS_NOT_SAVED',
-};
-
 type IEmbedTab = IModalTabDeclaration<{ url: string; isNotSaved: boolean }>;
 
-const embedTabReducer: IEmbedTab['reducer'] = (state = { url: '', isNotSaved: false }, action) => {
-  switch (action.type) {
-    case EMBED_TAB_ACTIONS.SET_IS_NOT_SAVED:
-      return {
-        ...state,
-        isNotSaved: action.payload,
-      };
-    case EMBED_TAB_ACTIONS.SET_IS_NOT_SAVED:
-      return {
-        ...state,
-        isNotSaved: action.payload,
-      };
-    default:
-      return state;
-  }
-};
-
 const EmbedTabContent: NonNullable<IEmbedTab['content']> = ({ state, dispatch }) => {
   const { embedUrlParamExtensions, shareableUrlForSavedObject, shareableUrl, objectType, isDirty } =
     useShareTabsContext()!;
 
-  const setIsNotSaved = useCallback(() => {
-    dispatch({
-      type: EMBED_TAB_ACTIONS.SET_IS_NOT_SAVED,
-      payload: objectType === 'dashboard' ? isDirty : false,
-    });
-  }, [dispatch, objectType, isDirty]);
-
   return (
     <EmbedContent
       {...{
         embedUrlParamExtensions,
         shareableUrlForSavedObject,
         shareableUrl,
         objectType,
-        isNotSaved: state?.isNotSaved,
-        setIsNotSaved,
+        isDirty,
```

Comment:
I was not sure about this. seems like we didnt pass it before and don't have in the original pr

Reply:
You were right... this resulted from conflict resolution, I've reverted this